packages/components/src/components/Forms/CheckboxGroup/CheckboxGroup.stories.tsx

```diff
@@ -0,0 +1,113 @@
+import { Meta, StoryObj } from '@storybook/react'
+import { View } from 'react-native'
+import React, { useState } from 'react'
+
+import { CheckboxGroup, CheckboxGroupProps } from './CheckboxGroup'
+import { generateDocs } from '../../../utils/storybook'
+
+const meta: Meta<CheckboxGroupProps> = {
+  title: 'Checkbox group',
+  component: CheckboxGroup,
+  decorators: [
+    (Story) => (
+      <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+        {Story()}
+      </View>
+    ),
+  ],
+  parameters: {
+    design: [
```

Comment:
Just adding a note that we've applied these very inconsistently on existing components and looking like one old case, Segmented control, has broken over time as Figma content got shuffled around.

@jessicawoodin, did you care if the Storybook pointed to Figma? I wonder if we're getting a bit duplicative here as Storybook does point to the doc site which includes both Storybook and Figma. General worries about ongoing maintenance as we know _someday_ Figma should move to VA's instance and this would be another spot to update.

For reference, but [Button is working to demo](https://department-of-veterans-affairs.github.io/va-mobile-library/?path=/story/button--primary) how it appears as the "Design" tab in the bar with the controls/actions:
![image](https://github.com/user-attachments/assets/761f79da-e2c5-431a-8dca-7b478b090360)